src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -208,12 +215,19 @@ object EasyIngest {
       System.exit(13)
     }
 
+  private class CompositeException(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)} ${getStackTrace(t)}"))
+  private class CompositeExceptionMessages(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)}"))
   private implicit class ListTryExtensions[T](xs: List[Try[T]]) {
     def sequence: Try[List[T]] =
       if (xs.exists(_.isFailure))
         Failure(new CompositeException(xs.collect { case Failure(e) => e }))
       else
         Success(xs.map(_.get))
-  }
 
+    def squashedSequence: Try[List[T]] =
```

Comment:
@jo-pol @rvanheest :
Proposing two improvements:
- Make inclusion of the stack traces configurable with a boolean
- Rename `sequence` to `collectResults` which IMO describes much better what it does.
See: https://github.com/jo-pol/easy-ingest/pull/1
See: https://github.com/jo-pol/easy-ingest/pulls